python/llm/example/GPU/LangChain/transformer_int4_gpu/README.md

```diff
@@ -0,0 +1,25 @@
+# Langchain examples
+
+The examples here show how to use langchain with `bigdl-llm`.
```

Comment:
The examples in this folder shows how to use [LangChain] ( add hyper link) with `bigdl-llm` on Intel GPU.